cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -389,6 +390,20 @@ func IsDeletionFailedBefore(cStorPool *apis.CStorPool) bool {
 	return false
 }
 
+// syncUsedCapacity generate warning if used capacity reach the default limit
+func (c *CStorPoolController) syncUsedCapacity(cStorPool *apis.CStorPool) {
+	if pool.UsedCapacity >= common.UsedCapacityLimitEightyPercent {
```

Comment:
Is there any possiblity of generating more such events for other % values like 50%,60%,40%? If yes then it would be better to modify this function to remove redundant code. If not you can ignore this comment.